src/Symfony/Bridge/Twig/Resources/views/Form/form_tailwind_layout.html.twig

```diff
@@ -0,0 +1,60 @@
+{% extends 'form_div_layout.html.twig' %}
```

Comment:
If it is for tailwind v2, what do you think of renaming the file with 2 inside, like bootstrap ones?

Reply:
That sounds reasonable.